packaging/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: skipper eskip docker-push
```

Comment:
yup, just then if you change something in the code, make may not detect that it needs to rebuild the binaries. Is there way to tell make that some source has changed? something like:

```
skipper: ../**/*.go
```

for me it didn't work by default. Any suggestions?